config/mailcoach-sdk.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    /*
+     *  You'll find both the API token and endpoint on Mailcoach'
+     *  API tokens screen in the Mailcoach settings.
+     */
+    'api_token' => env('MAILCOACH_API_TOKEN'),
```

Comment:
I don't see this token is being used, probably you need to add `withToken(config('mailcoach-sdk.api_token'))` in the Http request above. I would also move these to `mail.php` file so it is more clear and in that case you could use `withToken(config('mail.mailcoach.api_token'))`

Reply:
I used the Mailcoach package, which provides all the necessary features.